src/components/EditorMarkdownIt.vue

```diff
@@ -29,7 +33,7 @@ export default {
 		})
 
 		md.use(require('markdown-it-task-checkbox'), {
-			disabled: true,
+			disabled: false,
```

Comment:
```suggestion
			disabled: this.readonly,
```

Reply:
It seems this has to be inverted, but i have added this!

Reply:
No, it must not be inverted. With your change, it doesn't work anymore :wink:

Reply:
Yeah, you're right. Having like 20 notes all called "new note" does not help distinguising which one was shared and which ones were not. I was wondering why i had to invert it :facepalm: